app/Http/Controllers/CampaignRoleController.php

```diff
@@ -151,11 +168,13 @@ public function update(StoreCampaignRole $request, CampaignRole $campaignRole)
      * @param  \App\Models\CampaignRole  $campaignRole
      * @return \Illuminate\Http\Response
      */
-    public function destroy(CampaignRole $campaignRole)
+    public function destroy(CampaignRole $campaignRole, bool $bulk = false)
```

Comment:
Feels "wrong" to have a controller function being called by another controller function. Since all we're doing it $campaigRole->delete(), we can call that directly in the bulk process instead of complicating the controller's destroy function.